composer/utils/eval_client/eval_client.py

```diff
@@ -18,13 +18,11 @@ class EvalClient(abc.ABC):
     def invoke(self, payload: Dict[str, str]) -> bool:
         """Invoke a provided dictionary payload to the client.
 
-        For code generation, the payload has format:
-        {
+        For code generation, the payload is a JSON with the following attributes:
```

Comment:
I would strongly recommend seeing if we can add `{}`... but it's ok if you really think it's hard with linter

Reply:
fixed